src/convert.c

```diff
@@ -555,6 +555,45 @@ u8 base64_to_int (const u8 c)
   return tbl[c];
 }
 
+// alternate base64 using ./ instead of +/, used in python passlib hashes
+u8 int_to_alternate_base64 (const u8 c)
```

Comment:
int_to_ab64 looks more streamlined than int_to_alternate_base64 in comparison with the other existing buffers. Or something involving substring 'passlib' ?